CHANGELOG.md

```diff
@@ -1,22 +1,36 @@
 # astro changelog
 
-## 0.5.0 (UNRELEASED, 2019)
+## 0.5.0 (October 3, 2019)
 
+### Added
+* Add ls command to tvm (#37)
 * Add `version` command (#12)
 * Add binaries via goreleaser (#14)
+* Support detach flag with Terraform 0.12 (#45)
+* Make plan output work with terraform 0.12 (#41)
```

Comment:
These can just say "Support Terraform 0.12" and put it at the top, it's the biggest change!

Reply:
Done, yeah that makes sense. Sorry I didn't put a lot of though into making this changelog in the first run.